strata.default.lua

```diff
@@ -31,78 +31,70 @@ strata.set_bindings {
-	{
-		triggers = { event = "win_open_pre", class_name = "firefox" },
-		action = function(window) window.send_to_workspace(1) end,
-	},
-
-	-- A single action that can be ran from multiple triggers.
-	{
-		triggers = {
-			-- Set mpv to floating, always
-			{ event = "win_open_pre", class_name = "mpv" },
-			-- Set terminals to floating only if on the workspace 1 (where we put firefox previously)
-			{ event = "win_open_pre", workspace = 1, class_name = { "kitty", "wezterm" } },
-		},
-		action = function(window) window.set_floating() end,
-	},
-
-	-- Here comes the nice stuff: helper functions! This is important, as these functions help make the config file much
-	-- easier to write for the majority of users. Ideally, most people won't ever need to write something detailed like
-	-- what's above.
+-- strata.set_rules {
+-- 	-- Always bind firefox to the workspace 1, before the window is opened
+-- 	{
+-- 		triggers = { { event = "win_open_pre", class_name = "firefox" } },
+-- 		action = function(window) window.send_to_workspace(1) end,
+-- 	},
+--
+-- 	-- A single action that can be ran from multiple triggers.
+-- 	{
+-- 		triggers = {
+-- 			-- Set mpv to floating, always
+-- 			{ event = "win_open_pre", class_name = "mpv" },
+-- 			-- Set terminals to floating only if on the workspace 1 (where we put firefox previously)
+-- 			{ event = "win_open_pre", workspace = 1, class_name = "kitty" },
+-- 		},
+-- 		action = function(window) window.set_floating() end,
+-- 	},
 
-	-- This one does the same thing as the first rule
-	strata.rules.bind_to_workspace(1, "firefox"),
-
-	-- Nothing preventing us from some more sugar, to make things even less verbose:
-	strata.rules.bind_to_workspace {
-		{ 1, "firefox" },
-		{ 2, "neovide" },
-		{ 10, "slack" },
-	},
+-- Here comes the nice stuff: helper functions! This is important, as these functions help make the config file much
+-- easier to write for the majority of users. Ideally, most people won't ever need to write something detailed like
+-- what's above.
 
-	-- This does the same thing as the first trigger of the second rule
-	strata.rules.set_floating("mpv"),
-}
+-- This one does the same thing as the first rule
+-- strata.rules.bind_to_workspace(1, "firefox"),
+--
+-- -- Nothing preventing us from some more sugar, to make things even less verbose:
+-- strata.rules.bind_to_workspace({
+-- 	{ 1, "firefox" },
+-- 	{ 2, "neovide" },
+-- 	{ 10, "slack" },
+-- }),
+--
+-- -- This does the same thing as the first trigger of the second rule
+-- strata.rules.set_floating("mpv"),
+-- }
 
 -- Same as above, this is a function call. I think as it stands it should *overwrite* the whole config every time it's
 -- called. For a "merge" behavior, we could have a `strata.update_config()` function instead. This can one day be useful
 -- for plugins.
 strata.set_config {
 	autostart = {
-		{ "kitty", "--title", "Terminal" },
+		{ "kitty --title Terminal" },
```

Comment:
It's fine that we support this, but it's important to *also* support the decomposed way, so that arguments containing spaces can be handled without trouble.